src/reducers/dataLoggerReducer.js

```diff
@@ -69,11 +70,17 @@ export const updateDurationSeconds = durationSeconds => ({
     durationSeconds,
 });
 
+export const setSamplingAttrsAction = maxContinuousSamplingTimeUs => ({
+    type: SET_SAMPLING_ATTRS,
+    payload: {
+        maxContinuousSamplingTimeUs,
+    },
```

Comment:
It'd be simpler not to wrap the value in payload.

Reply:
I think it's cleaner to use the payload wrapper. That way it's easy to know what's on the action object, it's type and payload. And payload contains all the data that will be used in the updated state. It's not as straightforward here since we are not using this convention elsewhere in the app atm, so I'll update it according to your comment for now, but in the future it's something we should consider at least.

That's also how they use action in the official docs, e.g. from https://redux.js.org

```
const addTodo = text => {
  return {
    type: 'todos/todoAdded',
    payload: text
  }
}
```